promql/functions.go

```diff
@@ -189,3 +677,40 @@ func GetFunction(name string) (*Function, bool) {
 	function, ok := functions[name]
```

Comment:
Does `GetFunction` still need to be exported in the new code, since it's now all in one package?